browser/resources/settings/email_aliases_page/content/onEnterKey.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
```

Comment:
shouldn't this filename use underscores?

Reply:
fixed